lib/tezos/lib/assets/copy-data-to-s3.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set +e
+
+source /etc/environment
```

Comment:
We don't seem to be stopping the node before copying files. Can it lead to corrupted files being copied (unclosed, partially written, etc.)?

Reply:
The script now stops the service before syncing.